Sources/NIOSSL/SSLPKCS12Bundle.swift

```diff
@@ -158,6 +158,86 @@ public struct NIOSSLPKCS12Bundle: Hashable {
 
 extension NIOSSLPKCS12Bundle: Sendable {}
 
+extension NIOSSLPKCS12Bundle {
+    /// Create a PKCS#12 file containing the given certificates.
+    ///
+    /// The first certificate of the `certificates` array will be considered the "primary" certificate for
+    /// this PKCS#12, and `privateKey` must be its corresponding private key.
+    /// The other certificates included in `certificates`, if any, will be considered as additional
+    /// certificates in the certificate chain.
+    ///
+    /// - Parameters:
+    ///   - certificates: An array of certificates to include in this PKCS#12. Must contain at least one certificate.
+    ///   - privateKey: The private key associated to the first certificate in `certificates`.
+    ///   - passphrase: The password with which to protect this PKCS#12 file.
+    ///   - name: The name to give this PKCS#12 file.
+    /// - Returns: An array of bytes making up the PKCS#12 file.
+    public static func makePKCS12<Bytes: Collection>(
+        certificates: [NIOSSLCertificate],
+        privateKey: NIOSSLPrivateKey,
+        passphrase: Bytes,
+        name: Bytes
+    ) throws -> [UInt8] where Bytes.Element == UInt8 {
```

Comment:
I think this API does too much. Should we split it out? We can have a memberwise constructor on the PKCS12, and then a serialize function.

Reply:
Do you mean having an `init(certificateChain: [NIOSSLCertificate], privateKey: NIOSSLPrivateKey)` on `NIOSSLPKCS12Bundle`, and then a `serialize(passphrase: Bytes) -> [UInt8]`?

Reply:
Yeah.